src/interpolated-path.cc

```diff
@@ -126,6 +126,31 @@ namespace hpp {
       return true;
     }
 
+    void InterpolatedPath::impl_derivative
+    (vectorOut_t result, const value_type& param, size_type order) const
+    {
+      assert (param >= timeRange().first);
+      if (param == timeRange ().first || timeRange ().second == 0) {
+	result.noalias () = initial();
+	return;
+      }
+      if (param >= timeRange ().second) {
+	result.noalias () = end();
+	return;
+      }
```

Comment:
Are `dp/dt (0) != p(0)` and `dp/dt (1) != p(1)` ?
Otherwise, the 8 lines above have not been updated.